gourmet/plugin_gui.py

```diff
@@ -61,6 +62,17 @@ def make_list_store (self, plugin_list):
                 )
         return ls
 
+    @staticmethod
+    def plugin_description_formatter(col: Gtk.TreeViewColumn, renderer: Gtk.CellRendererText,
+                                     mod: Gtk.ListStore, itr: Gtk.TreeIter, data: Optional[Any]) -> None:
```

Comment:
`Optional[Any]` is redundant as `None` is already part of `Any`. Otherwise great